force-app/main/default/lwc/attendance/attendance.html

```diff
@@ -8,6 +8,19 @@
   -->
 
 <template>
+    <lightning-card if:true={displayHeader}>
+        <div class="slds-var-m-left_medium">
+            <lightning-record-form
+                record-id={recordId}
+                object-api-name={serviceSession}
+                layout-type="Compact"
+                mode="readonly"
```

Comment:
Food for thought not for this iteration but I wonder if we should add breadcrumbs to make it easier to get back to the session since the Name is not clickable in this context.

Reply:
Not sure if we need that since we are opening it in a new tab and they have the old tab to go back to. Let me know your thoughts?

Reply:
Oh yea! That makes sense :)